### ISSUE TITLE
CommandPalette's "More Options" chevron is ignored by screen readers

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#9247]
# Environment

WT version: after #9234 merges

# Steps to reproduce

1. open command palette
2. Navigate focus to a command with more options '>'

# Expected behavior

The screen reader should probably read that as "more options" or something like that.

# Actual behavior

Chevron ignored.

**Comment by DHowett** [comment]
You'll be VERY surprised that this is a /dup of #7908

> State should announce state as Collapsed or it should provide extra role description like "Has pop Up" which helps in differentiating the New tab list items and also the purpose of New tab having sub menu.

Reading over this report -- it is reporting that command palette _submenu items_ do not have call-outs!
